coalib/bearlib/languages/definitions/ResourceBundle.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ResourceBundle:
+    extensions = '.txt',
```

Comment:
we should have something like

```python
@Language
class PlainText:
    extensions = '.txt'
```

as well IMO and it's not the definitions that should be "aware" of this but the algorithms that operate on them, they should deal with overlaps etc